protobufs/livekit_sip.proto

```diff
@@ -232,6 +232,8 @@ message SIPOutboundTrunkInfo {
   // Map SIP X-* headers from 200 OK to SIP participant attributes.
   // Keys are the names of X-* headers and values are the names of attributes they will be mapped to.
   map<string, string> headers_to_attributes = 10;
+
+  bool krisp_enabled = 11;
```

Comment:
can we set this on the CreateSIPParticipant API instead? For outbound calls, it would give better control if it could be enabled or disabled for each call.

for inbound calls, since there's no explicit APIs, we'd have to leave any customizable options on the trunk